js/start.js

```diff
@@ -496,38 +497,16 @@ function ensureValidSettingsCurrency() {
 // let's start our flow - do we need onboarding?,
 // fetching app-wide models...
 function start() {
+  // This is the server config as returned by ob/config. It has nothing to do with
+  // app.serverConfigs which is a collection of server configuration data related
+  // to connecting with a server. The latter is stored in local storage.
+  // TODO - instead of these elaborate comments explaining the distinction, perhaps rename
+  // serverConfigs to serverConnectionConfigs?
   fetchConfig().done((data) => {
-    // This is the server config as returned by ob/config. It has nothing to do with
-    // app.serverConfigs which is a collection of server configuration data related
-    // to connecting with a server. The latter is stored in local storage.
     app.serverConfig = data || {};
-
-    if (!isCryptoCurrencySupported(app.serverConfig.cryptoCurrency)) {
-      const connectLink =
-        '<button class="btnAsLink js-connect clrTEm">' +
-          `${app.polyglot.t('unsupportedCryptoCurDialog.connectLink')}` +
-          '</button>';
-
-      const unsupportedCryptoCurDialog = openSimpleMessage(
-        app.polyglot.t('unsupportedCryptoCurDialog.title'),
-        app.polyglot.t('unsupportedCryptoCurDialog.body', {
-          curCode: app.serverConfig.cryptoCurrency,
-          connectLink,
-        }),
-        {
-          dismissOnEscPress: false,
-          showCloseButton: false,
-        }
-      );
-
-      unsupportedCryptoCurDialog.$('.js-connect')
-        .on('click', () => app.connectionManagmentModal.open());
-
-      serverConnectEvents.once('connected', () => unsupportedCryptoCurDialog.remove());
-
-      return;
-    }
-
+    app.serverConfig.walletsHelp = 'CAUTION! - Rather than reading the wallet curs directly ' +
```

Comment:
Is this helpful at all? If you were exploring `app.serverConfig` in the JS console, would you notice this warning?

Reply:
When would you see this in the console?

Reply:
if a developer is examining `app.serverConfig` in the console. Rather than them write code that directly references `app.serverConfig.wallets`, the preference is for them to get the curs from a utility function